python/tests/resources/V2StachUtils.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
can you rename this file to something like v2StachWithCompressedColumn.json

Reply:
changed the name of the file to V2StachWithCompressedColumn.json